### PR TITLE
Hide color fields

### PR DESCRIPTION
**Pull request by qWici**
I think it would be nice to be able to hide fields that are not necessary for the user. For example, in my project I want to let the user select the color only with HEX code. 
I haven't found contributing guide so I'm not sure how to test and build this project correctly. If you need any other changes on this code - I will be happy to do it. And thanks for the great package

### CODE REVIEW

**Comment by Wondermarin** [comment]
Yes, I thought to implement it, but you beat me to it :)


Thank you for reminding me about the contributing guide, I will definitely add it, as well as the tests.

I'll check it out and merge it tomorrow.

File: src/utils/index.ts [hunk 1]
```diff
@@ -5,6 +5,6 @@ export { getColorByCoordinates } from "./getColorByCoordinates";
 export { getCoordinatesByColor } from "./getCoordinatesByColor";
 export { getHueByCoordinates } from "./getHueByCoordinates";
 export { getCoordinatesByHue } from "./getCoordinatesByHue";
-export { isValidHEX } from "./isValidHEX";
+export { isValidHEX } from "./isValidHex";
```
Comment (Wondermarin) [inline src/utils/index.ts:8]:
```suggestion
export { isValidHEX } from "./isValidHEX";
```
I think you should change the name of the file, not the name of its export.

File: src/utils/isValidHex/index.ts [hunk 2]
```diff
@@ -1 +1 @@
-export { isValidHEX } from "./isValidHEX";
+export { isValidHEX } from "./isValidHex";
```
Comment (Wondermarin) [inline src/utils/isValidHex/index.ts:1]:
```suggestion
export { isValidHEX } from "./isValidHEX";
```

**Review by Wondermarin — CHANGES_REQUESTED** [review]
I've tested and fixed some things, as soon as you change the file name I'll be ready to merge the branches.

Thank you for your help <3

File: src/utils/index.ts (hunk 1 above)
Comment (qWici) [reply src/utils/index.ts]: @Wondermarin fixed

**Review by Wondermarin — APPROVED** [review]
Thank you for your work, the minor update will be released today or tomorrow.

**Pull request merged**